tests/v2/validation/provisioning/hostnametruncation/README.md

```diff
@@ -12,43 +12,44 @@ Please note that the config can be in either JSON or YAML (all examples are illu
 ## Getting Started
 See below a sample config file to run this test:
 ```yaml
-rancher:
-  host: ""
-  adminToken: ""
-  clusterName: ""
+rancher:                                   
+  host: ""                                  #required
+  adminToken: ""                            #required
+  clusterName: ""                           
+
 provisioningInput:
-  machinePools:
-  - nodeRoles:
-      etcd: true
+  machinePools:                             
+  - nodeRoles:                              #required (at least 1)
+      etcd: true                            #required (at least 1 controlplane & etcd & worker)                            
       quantity: 1
   - nodeRoles:
       controlplane: true
       quantity: 1
   - nodeRoles:
       worker: true
       quantity: 1
-  rke2KubernetesVersion: ["v1.27.6+rke2r1"]
-  cni: ["calico"]
-  providers: ["linode"]
-linodeCredentials:
-   token: ""
-linodeMachineConfig:
-  authorizedUsers: ""
-  createPrivateIp: true
-  dockerPort: "2376"
-  image: "linode/ubuntu22.04"
-  instanceType: "g6-standard-8"
-  region: "us-west"
-  rootPass: ""
-  sshPort: "22"
-  sshUser: ""
-  stackscript: ""
-  stackscriptData: ""
-  swapSize: "512"
-  tags: ""
-  uaPrefix: "Rancher"
+  rke2KubernetesVersion: ["v1.27.6+rke2r1"] #required (at least 1)
+  cni: ["calico"]                           #required (at least 1)
+  providers: ["aws"]                        #required (at least 1)
+
+awsCredentials:                            
+  secretKey: "",                            #required                                               
+  accessKey: "",                            #required                          
+  defaultRegion: ""                         #required                      
+
+awsMachineConfig:                                   
```

Comment:
this is now 
```
awsMachineConfigs:
  - awsMachineConfig:...
```

Reply:
I just rebased again and picked that up.

Reply:
looks like it isn't picket up?